src/Commands/DocsCommand.ts

```diff
@@ -22,6 +22,12 @@ export class DocsCommand extends CommandBase {
 
     }
 
+    public static getEmbedMsg(title: string, color: number, description: string): RichEmbed {
```

Comment:
This function may want to be moved outside of the file? Other commands seem to want to use this structure a lot so it may be fine to add as a library helper.

Reply:
yes, perfect!